.gitignore

```diff
@@ -5,3 +5,8 @@ bin/
 .DS_Store
 
 vendor/
+
+e2e/opentofu/.terraform/**
```

Comment:
Just FYI, in other projects we use Linode Cluster API because Terraform provider is deprecated as i know.